### DISCUSSION TITLE
Any practical calculator for Mac

### DISCUSSION THREAD

**Discussion by M-Amrollahi** [Homebrew/discussions#133] [Everyday usage]
Hello,
I have not found any calculator which includes History. The default on MacOS does not have this feature and I think this is necessary.
Is there any simple but practical for Mac like [Moffsoft](http://www.moffsoft.com/freecalc.htm)

**Comment by SMillerDev** [comment]
Homebrew isn't really a support forum for macOS users. https://apple.stackexchange.com/ might be the better option

**Reply by M-Amrollahi** [reply]
I do not need support. Just asking for any better calculator app for MacOs instead the default one...

**Reply by SMillerDev** [reply]
And we don't answer those kinds of questions because it's not homebrew related

**Reply by M-Amrollahi** [reply]
So, just close it.

**Comment by jonchang** [comment]
Try running `brew search --desc calculator`, which will search both formulae and casks (on macOS) and give you a few things to see if they fit your needs.

**Reply by M-Amrollahi** [reply]
Thanks Jonathan,
The best way was to search in descriptions.🙏

**Comment by EricFromCanada** [comment]
Also, the macOS Calculator app has history under _Window > Show Paper Tape_.
